jest.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
             '<rootDir>/internals/mocks/fileMock.js',
         '\\.(css|less|sass|scss)$': 'identity-obj-proxy',
         '^~/(.*)$': '<rootDir>/app/$1',
+        '@concordium/web-sdk': '@concordium/common-sdk',
```

Comment:
Why is this needed?

Reply:
I don't remember, maybe because of the wasm module, but tests just don't work directly on web-sdk, I had to do the same for BW.